src/permission/fs_permission.cc

```diff
@@ -17,12 +17,20 @@
 namespace {
 
 std::string WildcardIfDir(const std::string& res) noexcept {
-  auto path = std::filesystem::path(res);
-  auto file_status = std::filesystem::status(path);
-  if (file_status.type() == std::filesystem::file_type::directory) {
-    path /= "*";
+  uv_fs_t req;
```

Comment:
@joyeecheung Is it safe to keep this function, or do we need this revert?

Reply:
Is `std::filesystem::path(res)` correct? From what I can tell, `res` contains UTF8-encoded data, and it's about to be passed into `uv_fs_stat` which also expects UTF8-encoded data. So the code here is also going to crash on Windows when `res` contains code points that are encoded differently in UTF16.